packages/modeling/src/geometries/geom2/reverse.js

```diff
@@ -12,11 +11,9 @@ import toSides from './toSides.js'
  * let newgeometry = reverse(geometry)
  */
 export const reverse = (geometry) => {
-  const oldsides = toSides(geometry)
-
-  const newsides = oldsides.map((side) => [side[1], side[0]])
-  newsides.reverse() // is this required?
-  return create(newsides)
+  const reversed = clone(geometry)
+  reversed.outlines.forEach((outline) => outline.reverse())
```

Comment:
array.reverse() does an 'in-place' reversal of the contents. i think this will change the original geometry as well.